src/shared/utils.ts

```diff
@@ -212,6 +212,22 @@ export function escapeHTML(
     return output;
 }
 
+/**
+ * Returns a string containing the label and readable keyboard shortcut for button tooltips
+ * @param label The type to toggle to
+ * @param mapping? Corresponding command mapping (keyboard shortcut)
+ */
+export function getShortcut(mapping?: string): string {
+    if (mapping.indexOf("Mod-") !== 0) {
+        return mapping;
+    }
+
+    return mapping.replace(
+        "Mod-",
+        /Mac/.test(navigator.userAgent) ? "Cmd-" : "Ctrl-"
```

Comment:
Fwiw, here's the regex that `prosemirror-keymap` uses to test for macOS support:

https://github.com/ProseMirror/prosemirror-keymap/blob/master/src/keymap.ts#L5